eng/Packages.Data.props

```diff
@@ -129,6 +129,7 @@
     <PackageReference Update="Azure.Storage.Queues" Version="12.19.1" />
     <PackageReference Update="Azure.Storage.Files.Shares" Version="12.19.1" />
     <PackageReference Update="Azure.AI.OpenAI" Version="2.0.0" />
+    <PackageReference Update="Azure.AI.Inference" Version="1.0.0-beta.2" />
```

Comment:
Can we bump these up so that the list in this section  is alphabetical by package name ?
```suggestion
    <PackageReference Update="Azure.AI.Inference" Version="1.0.0-beta.2" />    
    <PackageReference Update="Azure.AI.OpenAI" Version="2.0.0" />
```